llm-server/src/llm-provider.ts

```
@@ -1,79 +1,161 @@
+        try {
+          // Determine model type
+          const type = this.determineModelType(modelConfig);
+
+          // Create model instance directly
+          const modelKey = modelConfig.alias || modelConfig.model;
+          switch (type) {
+            case 'remote':
+              const instance = await RemoteModelFactory.createInstance(
+                modelConfig,
+                modelConfig.model,
+              );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)